future/name-resolution/002-nr-host-synchronization/README.md

```diff
@@ -412,19 +438,19 @@ Each arrow in this diagram indicates a new proxied request for name resolution f
 
 Two important things to note about the request flow in this diagram:
 
-First, when a router receives a request on one interface, it doesn't then send another proxied request out that interface, e.g. `router-5` doesn't proxy a request back to `router-4`.
+* First, when a router receives a request on one interface, it doesn't then send another proxied request out that interface, e.g. `router-5` doesn't proxy a request back to `router-4`.
 
-Second, when a router receives a duplicate request, e.g. `router-5` receives a request from both `router-4` AND `router-1`, it doesn't start another round of proxying for that request.
+* Second, when a router receives a duplicate request, e.g. `router-5` receives a request from both `router-4` AND `router-1`, it doesn't start another round of proxying for that request.
 
-The result of these behaviors is that messages don't go around the internet forever. Each message gets to every network only one time.
+The result of these behaviors is that messages don't go around the internet forever. Each message gets to every network once and only once. This is what we mean by "flooding" the network.
 
-That shows us how the request has been flooded over the internet. Now we need to look at how the response makes its way to every machine as well.
+Next, let's look at how the response makes its way to every machine as well.
 
 ![flowchart of a proxy response through our internet](../img/proxy-response.svg)
 
-Here, we see that `host-h` generates the initial message that then hits every router on `4.0.0.0/8`. Each router proxies that message to each machine on each network it has an interface on. This allows the response to reach every machine on our internet.
+Here, we see that `host-h` generates the response message that gets broadcast to every router on the `4.0.0.0/8` network. Each router then proxies that message to each machine on each other network it has an interface on. As each router, in turn, does the same thing, this allows the response to flood back to every machine on the internet.
 
-What this means is that every machine on our internet will already have an answer for where to find `host-h.local`. So, if, after the name resolution has been performed and during the cache window, if you run a `tcpdump` on a different host while you run `ping host-h.local` on that host, you won't see any name resolution traffic.
+What this means is that every machine on our internet will already have an answer for where to find `host-h.local`. So, if, after the name resolution has been performed and during the cache window, if you run a `tcpdump` on a different host while you run `ping host-h.local` on that host, you won't see any name resolution traffic, because it already knows the IP address of that machine!
```

Comment:
```suggestion
What this means is that every machine on our internet will already have an answer for where to find `host-h.local`. So, after the name resolution has been performed and during the cache window, if you run a `tcpdump` on a different host while you run `ping host-h.local` on that host, you won't see any name resolution traffic because it already knows the IP address of that machine!
```